C# in-depth/006_Reflection/Reflection/HomeTask2/Program.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.IO;
+using System.Reflection;
+
+namespace HomeTask2
+{
+    class Program
+    {
+        static void Main()
+        {
+            Assembly assembly = null;
+
+            try
+            {
+                assembly = Assembly.Load("Temperature");
+                Console.WriteLine(assembly.FullName + " loaded");
+            }
+            catch (FileNotFoundException ex)
+            {
+                Console.WriteLine(ex.Message);
+            }
+
+            Console.WriteLine("What operation do you want to do?\n1.Convert celsius to fahrenheit.\n2.Convert fahrenheit to celsius.");
+            int operation = Convert.ToInt32(Console.ReadLine());
+            
+            switch (operation)
+            {
+                case 1:
+                    {
+                        Convertation(assembly, "ConvertToFahrenheit", "F");
+                        break;
+                    }
+                case 2:
+                    {
+                        Convertation(assembly, "ConvertToCelsius", "C");
+                        break;
+                    }
+                default:
+                    {
+                        Console.WriteLine("Command is invalid");
+                        break;
+                    }
+            }
+
+            Console.ReadKey();
+        }
+
+        static void Convertation(Assembly assembly, string operation, string degree)
```

Comment:
Does the method work correctly if we call it like that `Convertation(assembly, "ConvertToCelsius", "F")` ?